docs/UserGuide.md

```diff
@@ -7,14 +7,15 @@
 * **[3. Features](#features)**
   * [3.1 Adding an item: `ADD`](#adding-an-item-add)
   * [3.2 Editing an item: `EDIT`](#editing-an-item-edit)
-  * [3.3 Deleting an item: `Delete`](#deleting-an-item-del)
+  * [3.3 Deleting an item: `DEL`](#deleting-an-item-del)
```

Comment:
Thanks for spotting the error! I edited this line as well, but I realised I renamed to DELETE instead, my bad

Reply:
No probbies!! Changed it already. Thanks:)